zio-http/src/main/scala/zhttp/http/HttpData.scala

```diff
@@ -85,12 +87,20 @@ object HttpData {
   /**
    * Helper to create HttpData from contents of a file
    */
-  def fromFile(file: java.io.File): HttpData = File(file)
+  def fromFile(file: java.io.File): HttpData = {
+    val raf = new RandomAccessFile(file, "r")
+    File(raf, file.toPath)
+  }
+
+  def fromFileZIO(file: java.io.File): Task[HttpData] = Task {
+    val raf = new RandomAccessFile(file, "r")
+    File(raf, file.toPath)
+  }
```

Comment:
Constructors should return it's own type.

Reply:
Removed this.